defs.bzl

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 load("@rules_jvm_external//:coursier.bzl", "coursier_fetch")
-load("@rules_jvm_external//:specs.bzl", "maven", "parse", "json")
```

Comment:
was maven unused? I'm very impressed buildifier knows how to check for that

Reply:
Yep! Caught by buildifier